### PR TITLE
Added README for rod cutting problem

### PR DESCRIPTION
**Pull request by rimshaakhan**
**Fixes issue:** #3314 

**Changes:**
Added README for Rod Cutting problem.

### CODE REVIEW

File: code/dynamic_programming/src/rod_cutting/README.md [hunk 1]
```diff
@@ -1,4 +1,33 @@
...
+
+length (i) | 1|2|3|4|5|6|7|8|9|10  
+-----------|--|-|-|-|-|--|-|--|--|------------  
+price (p<sub>i</sub>)|1|5|8|9|10|17|17|20|24|30
+
+For a rod of length n = 4, maximum revenue is obtained by cutting the rod into two pieces each of length 2 and the maximum revenue obtained is 10.
+
+ 
+## Explanation
+This problem satisfies both properties of Dynamic programming and can be solved by the following recurrence relation.
```
Comment (arnavb) [inline code/dynamic_programming/src/rod_cutting/README.md]: This is very little explanation.

File: code/dynamic_programming/src/rod_cutting/README.md [hunk 2]
```diff
@@ -1,4 +1,33 @@
...
+
+
```
Comment (arnavb) [inline code/dynamic_programming/src/rod_cutting/README.md]: You forgot to list an algorithm as a list of steps according to [our guide](https://github.com/OpenGenus/cosmos/blob/master/guides/documentation_guide.md).

File: code/dynamic_programming/src/rod_cutting/README.md [hunk 3]
```diff
@@ -1,4 +1,33 @@
...
+### Recurrence Relation
+r<sub>n</sub> = max (p<sub>i</sub> + r<sub>n-i</sub>)   for all 1 <= i <= n
```
Comment (arnavb) [inline code/dynamic_programming/src/rod_cutting/README.md]: I personally don't like this and would rather use a latex image, such as this one:

Math

**Comment by rimshaakhan** [comment]
@arnavb I have the requested changes. Please Review. :smile:

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**